proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
yeah, i feel like it is a bit strange to require config updates on both proxy side and mixer side to get a quota to be effective.

even if the proxy always calls Quota(), it seems like it shouldn't have to specify the name of the quota that applies (and that mixer should be able to determine it at request time).

Otherwise, is the plan to have mixer config updates propagate to proxy on every quota override?